docs/book/src/developers/development.md

```diff
@@ -552,8 +552,8 @@ You can optionally set the following variables:
 | `AZURE_SSH_PUBLIC_KEY_FILE` | Use your own SSH key.                                                                                                                      |
 | `SKIP_CLEANUP`              | Skip deleting the cluster after the tests finish running.                                                                                   |
 | `KUBECONFIG`                | Provide your existing cluster kubeconfig filepath. If no kubeconfig is provided, `./kubeconfig` will be used.                                  |
-| `USE_CI_ARTIFACTS`          | Use a CI version of Kubernetes, ie. not a released version (eg. `v1.19.0-alpha.1.426+0926c9c47677e9`)                                      |
-| `CI_VERSION`                | Provide a custom CI version of Kubernetes. By default, the latest master commit will be used.                                              |
+| `KUBERNETES_VERSION`        | Desired Kubernetes version to test. You can pass in a definitive released version, e.g., "v1.24.0"; or use the most recent built, releasable CI bits of a particular release by passing in e.g. "release-1.24"; or you may use the very latest built CI bits by passing in "latest".
```

Comment:
does `release-1.24` actually work? FWIW in conformance test we use "stable-1.24" to indicate the latest released patch version of 1.24

Reply:
https://dl.k8s.io/release/stable-1.24.txt does the job, but we'd have to add that into ci-entrypoint

Reply:
```
$ curl -sSL https://dl.k8s.io/ci/latest-1.24.txt
v1.24.1-rc.0.29+1868a3a2eb9945
```

```
$ curl -sSL https://dl.k8s.io/release/stable-1.24.txt
v1.24.0
```

Do we want to add support to `ci-entrypoint.sh` for the 2nd pattern?

Reply:
I think that'd be pretty useful, can be its own PR if you think that's a scope creep though

Reply:
https://github.com/kubernetes/release/blob/9e1b9044533a09adeac1a6c4706b665db8cd5cd0/pkg/release/version_test.go#L141-L174

Reply:
in any case, let's remove `"release-1.24"` since it's not a thing and replace it with `"latest-1.24"`

Reply:
Done